plugins/rest/url/rest_url.go

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package url
+
+// Access list REST keys
+const (
+	// REST Acl IP prefix
+	restAclIP = "/v1/acl/ip"
+	// REST Acl IP example prefix
+	restAclIPExample = "/v1/acl/ip/example"
+	// REST Acl MACIP prefix
+	restAclMACIP = "/v1/acl/macip"
+	// REST Acl MACIP example prefix
+	restAclMACIPExample = "/v1/acl/macip/example"
+)
+
+// RestIPKey returns prefix used in REST to dump ACL IP config
+func RestIPKey() string {
+	return restAclIP
```

Comment:
I totally fail to see a point in making functions return constant. It just more than triples the amount of code... here instead of having 4 lines of constants (8 with comments) we need 12 lines (16 with comments) without any added value.

I propose we just keep exported constants for all of these and also for all of the prefixes in models, those named `xxxPrefix`. We can then keep functions for those that generate full key from parameters, those are mostly named `xxxKey`.